commit 195923c21686906851ae2cf3ce1c58e74be28e6f
Author: Cody Russell <cody@jhu.edu>
Date:   Thu Mar 12 09:56:24 2020 -0500

    [Mac] Possible fix for crash in find_nsview_at_pos()
    
    Possibly fixes VSTS #1080409

diff --git a/gdk/quartz/gdkevents-quartz.c b/gdk/quartz/gdkevents-quartz.c
index a12597d0f9..f22722f25d 100644
--- a/gdk/quartz/gdkevents-quartz.c
+++ b/gdk/quartz/gdkevents-quartz.c
@@ -740,11 +740,14 @@ find_nsview_at_pos (GdkWindowImplQuartz *impl, gint x, gint y)
   for (i = 0; i < n_subviews; ++i)
     {
       NSView* sv = [[view subviews] objectAtIndex:i];
-      NSRect r = [sv frame];
 
-      if (sv == impl->layer_view)
+      if (sv == NULL)
+        return;
+      else if (sv == impl->layer_view)
         continue;
 
+      NSRect r = [sv frame];
+
       if (![sv isHidden] &&
           r.origin.x <= x && r.origin.x + r.size.width >= x &&
           r.origin.y <= y && r.origin.y + r.size.height >= y)
